test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/PagedBufferedTextWriterTest.cs

```diff
@@ -83,6 +102,19 @@ static PagedBufferedStringWriterTest()
         }
 
         [Fact]
+        public void Write_CharArray_Bounded_Null()
+        {
+            // Arrange
+            var pool = new TestArrayPool();
+            var inner = new StringWriter();
+
+            var writer = new PagedBufferedTextWriter(pool, inner);
+
+            // Act & Assert
+            Assert.Throws<ArgumentNullException>(() => writer.Write(null, 0, 0));
```

Comment:
Pass the `paramName` argument i.e. `Assert.Throws<ArgumentNullException>("buffer", () => writer.Write(null, 0, 0));`.